### PR TITLE
UI-Session-1 Assignment 

### PR DESCRIPTION
**Pull request by meta-manoj-sharma**

### CODE REVIEW

File: UI-Session-1/UI-Session-1.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+					<td><label><b>Contact Number</b></label></td>
+					<td><input type="number" name="contact" placeholder="Enter Contact Number" required> </td>
+				</tr>
+			</table>
+			<input type="submit" value="Register"> <br> <br>
+		</form>
+	</div>
+
+	<div align="center" class="divisionTwo" id="vehicleForm">
+		<form action="">
```
Comment (jbhavya) [inline UI-Session-1/UI-Session-1.html]: Remove the action attribute if you have not given any value to it.

File: UI-Session-1/UI-Session-1.html [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+				<td><textarea rows="3" cols="22" value=""> </textarea> </td>
+      </tr>
+    </table>
+			<input type="submit"> <br>
+			<br>
+		</form>
+	</div>
+
+	<div align="center" class="divisionThree" id="feedbackForm">
+		<form action="">
```
Comment (jbhavya) [inline UI-Session-1/UI-Session-1.html]: Remove the action attribute if you have not given any value to it.

File: UI-Session-1/UI-Session-1.html [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+        <table>
+				<tr>
+          <td><label> <input type="radio" name="motorCyclePass" value="Rs.10">Daily - Rs.10</label></td>
+				</tr>
+				<tr>
+          <td><label> <input type="radio" name="motorCyclePass" value="Rs.200">Monthly - Rs.200 </label></td>	
+				</tr>
+				<tr>
+          <td>	<label> <input type="radio" name="motorCyclePass" value="Rs.1000">Yearly - Rs.1000 </label></td>
+        </tr>
```
Comment (jbhavya) [inline UI-Session-1/UI-Session-1.html]: Correct indentation here.

File: UI-Session-1/UI-Session-1.html [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+<html>
+<head>
+<title>Metacube Parking System</title>
+<link rel="stylesheet" type="text/css" href="css1.css">
+</head>
+<body style="background-image: url('pattern.jpg');">
+	<h1 id="main" align="center">Metacube Parking System</h1>
+	<hr>
+	<nav>
+		<a href="#employeeForm">Employee Registration</a>&nbsp;&nbsp; <a
```
Comment (jbhavya) [inline UI-Session-1/UI-Session-1.html]: Fix indentation here.

File: UI-Session-1/UI-Session-1.html (hunk 1 above)
Comment (meta-manoj-sharma) [reply UI-Session-1/UI-Session-1.html]: logic updated

File: UI-Session-1/UI-Session-1.html (hunk 2 above)
Comment (meta-manoj-sharma) [reply UI-Session-1/UI-Session-1.html]: logic updated

File: UI-Session-1/UI-Session-1.html (hunk 3 above)
Comment (meta-manoj-sharma) [reply UI-Session-1/UI-Session-1.html]: resolved

File: UI-Session-1/UI-Session-1.html (hunk 4 above)
Comment (meta-manoj-sharma) [reply UI-Session-1/UI-Session-1.html]: resolved

**Pull request merged**